packages/thirdweb/src/utils/encoding/helpers/role-bytes.ts

```diff
@@ -0,0 +1,14 @@
+import { keccak256 } from "../../hashing/keccak256.js";
+import { toBytes } from "../to-bytes.js";
+
+/**
+ * Generates a 256-bit hash of a given role string in bytes form using the keccak256 algorithm.
+ *
+ * @param {string} role - The role string to be converted into bytes and hashed.
+ * @returns {`0x${string}`} A 256-bit hash of the input role as a byte array.
+ * @example
+ *   const AdminRole = roleBytes("ADMIN_ROLE");
+ */
+export const roleBytes = (role: string): `0x${string}` => {
```

Comment:
need to export it publicly, you can do that by exporting it from /exports

likely under /exports/extensions/common.ts where the other role stuff is